aten/src/ATen/function_wrapper.py

```diff
@@ -1060,11 +1082,16 @@ def find_formal(formal_name, formals):
         # Factory methods are not dispatched over `Type`.
         if not is_factory_method:
             if option['deprecated']:
+                # Deprecated functions are always non-internal
```

Comment:
why?

Reply:
```
                # Deprecated functions are always non-internal,
                # because they need to be made available from Type
                # (the public interface) so that code like
                # tensor.type().arange(...) keeps working.  Once
                # we remove the deprecated functions, we can eliminate
                # these methods entirely.

```